src/sass/_contacts.scss

```diff
@@ -0,0 +1,168 @@
+.contacts{
+    font-family: 'DM Sans', sans-serif;
+
+    &__container{
+        padding-top: 20px;
+        background: var(--bottom-section-background);
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+
+        &-inside{
+            padding-top: 35px;
+            padding-left: 20px;
+            padding-right: 20px;
+            padding-bottom: 42px;
+        }
+    }
+
+    &__display--none{
+        display: none;
+    }
+
+    &__section{
+        width: 220px;
+        height: 427px;
+        background-color: var(--white);
+        box-shadow: 0px 8px 30px var(--pink-shadow);
+        border-radius: 24px;
+        margin-bottom: 20px;
+    }
+
+    &__header{
+        border: 1px solid var(--lightgrey);
+        border-top: transparent;
+        border-right: transparent;
+        border-left: transparent;
+    }
+
+    &__title{
+        font-weight: 500;
+        font-size: 12px;
+        line-height: 186%;
+        width: 80px;
+        height: 16px;
+        background-color: var(--yellow);
+        border-radius: 4px;
+        color: var(--white);
+        text-align: center;
+        padding-bottom: 20px;
+        margin-bottom: 20px;
+    }
+
+    &__title-second{
+        font-weight: 500;
+        font-size: 12px;
+        line-height: 186%;
+        width: 130px;
+        height: 16px;
+        background: var(--green);
+        border-radius: 4px;
+        text-align: center;
+        padding-bottom: 20px;
+    }
+
+    &__city{
+        font-weight: 500;
+        font-size: 12px;
+        line-height: 186%;
+        letter-spacing: 0.04em;
+    }
+
+    &__describtion{
+        width: 179px;
+        height: 36px;
```

Comment:
Znów unikałbym sztywnej wysokości w takich elementach tekstowych, bo opis może się zmienić, być krótszy lub dłuższy i co wtedy?

Reply:
no moim zadaniem było zrobić wersję na telefon, jest ona w tym wypadku stała przy zmianie szerokości, więc ustawiłem stałą szerokość
przy robieniu wersji na tablet i dekstop można to zmienić